lib/exe/css.php

```diff
@@ -337,6 +354,41 @@ function css_fixreplacementurls($replacements, $location) {
 }
 
 /**
+ * Wrapper for the files, content and mediatype for the event CSS_STYLES_INCLUDED
+ *
+ * @author Gerry Weißbach <gerry.w@gammaproduction.de>
+ *
+ * @param string $mediatype type ofthe current media files/content set
+ * @param array $files set of files that define the current mediatype
+ * @return array
+ */
+function css_filewrapper($mediatype, $files=array()){
+    return array(
+            'files'                 => $files,
+            'mediatype'             => $mediatype,
+            'encapsulate'           => in_array($mediatype, array('screen', 'print', 'speech')),
```

Comment:
We now have two places where the mediatypes are defined. Here and way up at line 39.

Could this be reversed? Like `$mediatype != 'all'`?